packages/web-react/src/components/Pagination/PaginationLinkNext.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { ElementType, ForwardedRef, forwardRef } from 'react';
+import { SpiritPaginationLinkNextProps } from '../../types';
+import PaginationLinkDirectional from './PaginationLinkDirectional';
+
+/* We need an exception for components exported with forwardRef */
+/* eslint no-underscore-dangle: ['error', { allow: ['_PaginationLinkNext'] }] */
+const _PaginationLinkNext = <E extends ElementType = 'a'>(
+  props: SpiritPaginationLinkNextProps<E>,
+  ref: ForwardedRef<HTMLButtonElement>,
+) => <PaginationLinkDirectional direction="next" label="Next" ref={ref} {...props} />;
```

Comment:
The label must be configurable, please use a prop with default. I know that `prop` will override this, but I think that usage of the default value will be more readable. Also are we sure that the `direction` can be overridden by `props` in this case (did not see types for this component yet)?

Reply:
fixed